dali/dfuplus/dfuplus.cpp

```diff
@@ -1035,28 +1035,12 @@ int CDfuPlusHelper::remove()
         return -1;
     }
 
-    const char* result = resp->getDFUArrayActionResult();
     StringBuffer resultbuf;
-    if(result != NULL && *result != '\0')
```

Comment:
@rpastrana  It will be better if dfuplus (and other tools like ecl) can send requests based on server version. It should be in a separate JIRA.